docs/chat-configuration/ui-customization/incoming-message/carousel.md

```diff
@@ -17,32 +17,34 @@ nav_exclude: true
 
 ---
 
-## Overview
-Displays a "carousel" typed incoming response. 
-As a regular incoming message, carousel has a message section with a timestamp and avatar, and the quick options and channels at the bottom. The carousel provides the ability to display multiple related answers to a user query in a horizontal scrollable structure.
+## Overview 
+Displays the response as a carousel of items. 
+The carousel, just like a regular incoming message, has a message section with a timestamp and avatar, and the quick options and channels at the bottom. The carousel provides the ability to display multiple related answers to a user query in a horizontal scrollable structure.
 {: .overview}
 ![]({{'/assets/images/carousel.png' | relative_url}})
 {: .image-40}
 
 
 ## Carousel Item
-Each item in the carousel consists of 3 sections: 
-- **Image** - If not configured, a placeholder default image will be displayed.
-    Can be configured with a url link. <sub>Optional</sub>.
-- **Info** - Title <sub>Mandatory</sub> and a subtitle <sub>Optional</sub> 
-- **Options** - Item options <sub>Optional</sub>.
+Each corousel item consists of three display properties: 
+- **Image** <sub>(optional)</sub> - THe image can be configured with a URL link. If not image is set, a default image will be displayed.     
+- **Info** <sub>(optional)</sub> - Title <sub>(mandatory)</sub> and a subtitle for the item displayed.
```

Comment:
optional applies to the sub-title, the main title is mandatory

Reply:
OK